WordPress/Classes/Extensions/UIAlertController+Helpers.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+
+extension UIAlertController
+{
+    public func addCancelActionWithTitle(title: String?, handler: ((UIAlertAction) -> Void)?) -> UIAlertAction {
+        return addActionWithTitle(title, style: .Cancel, handler: handler)
+    }
+    
+    public func addDestructiveActionWithTitle(title: String?, handler: ((UIAlertAction) -> Void)?) -> UIAlertAction {
+        return addActionWithTitle(title, style: .Destructive, handler: handler)
+    }
+
+    public func addDefaultActionWithTitle(title: String?, handler: ((UIAlertAction) -> Void)?) -> UIAlertAction {
+        return addActionWithTitle(title, style: .Default, handler: handler)
+    }
+    
+    public func addActionWithTitle(title: String?, style: UIAlertActionStyle, handler: ((UIAlertAction) -> Void)?) -> UIAlertAction {
+        let action = UIAlertAction(title: title, style: style, handler: handler)
+        addAction(action)
+        
+        return action
+    }
+    
+    public func presentFromRootViewController() {
+        // Note:
+        // This method is required because the presenter ViewController must be visible, and we've got several
+        // flows in which the VC that triggers the alert, might not be visible anymore.
+        //
+        guard let rootViewController = UIApplication.sharedApplication().delegate?.window??.rootViewController else {
+            print("Error loading the rootViewController")
+            return
+        }
+        
+        var leafViewController = rootViewController
+        while leafViewController.presentedViewController != nil {
+            leafViewController = leafViewController.presentedViewController!
+        }
```

Comment:
@jleandroperez do you think this could fail with other view controller hierarchies? In the past this has always been fairly complicated given navigation controllers and additional presented view controllers. This might not be guaranteed to work in upcoming iOS versions, but I suppose there may not be a guaranteed way at all.

I tested this with different view controller/navigation controller/presented controller setups and it worked in each. There may be some extra logic within `UIAlertController` taking care of this for us.